### DISCUSSION TITLE
Blackhole output no volume control

### DISCUSSION THREAD

**Discussion by phen0** [ExistentialAudio/BlackHole#344] [General]
Hi,
I'm trying to get blackhole 16ch working. To install I followed exactly your description but if I assign the output device I can't control the volume any longer. Any clue what's the issue? I installed BlackHole16ch.v0.2.9.
Thanks

**Comment by devinroth** [comment]
https://github.com/ExistentialAudio/BlackHole#how-can-i-change-the-volume-of-a-multi-output-device
